include/boost/geometry/strategies/spherical/side_by_cross_track.hpp

```diff
@@ -71,9 +71,19 @@ public :
         calc_t crs_AD = geometry::formula::spherical_azimuth<calc_t, false>
                              (lon1, lat1, lon, lat).azimuth;
 
+        if (crs_AD == 0)
+        {
+            return 0;
+        }
+
```

Comment:
This doesn't look correct. Azimuth 0 means going north which doesn't mean that the point lies at the same great circle as the segment. Or am I missing something?

Reply:
This is a leftover from testing this function. I think the whole file could be removed, since this strategy is not used anywhere and `spherical_side_formula` does the same computation more efficiently.

Reply:
>This is a leftover from testing this function.

Ok, so these lines should be removed, right?

> I think the whole file could be removed, since this strategy is not used anywhere and `spherical_side_formula` does the same computation more efficiently.

Well, I don't know. One is using azimuths and the other 3d vectors and AFAIR the accuracy is different in both cases.

Reply:
OK, so let's keep it. However, since there is an issue which is similar to the geographic side issue I updated  https://github.com/boostorg/geometry/issues/765 and I propose here a similar fix to the one of the geographic side.